datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -62,6 +115,28 @@ def test_assign():
     eq_(list(gen(datadup)), [{'x': 'value', 'g': 'y z'}])
 
 
+def test__act_if():
+    values = {'x': 'y', 'a': 'b'}
+    data = {'x': 'y'}
+    datamatch = {'x': 'y', 'a': 'b'}
+
+    # matched = false
+    gen = _act_if(values, False, True)
```

Comment:
btw -- `_act_if` is not to be used directly -- it is a base class for `interrupt_if` and `skip_if` and theoretically if any match should raise NotImplementedError from its `_act`.  (eventually -- look into abc module and how to use `@abstractmethod` which should ideally be done for `_act_if._act`

So if testing `_act_if`, tests should raise NotImplementedError when "appropriate"